modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -494,10 +498,13 @@
 			// Return a "textSource" object:
 			return new mw.TextSource( embedSource, _this.embedPlayer );
 		},
-		forceLoadLanguage: function(){
-			var lang = this.getConfig('forceLoadLanguage');
+		forceLoadLanguage: function(language){
+			var lang = this.getConfig('forceLoadLanguage') ? this.getConfig('forceLoadLanguage') : language;
 			var source = this.selectSourceByLangKey( lang );
-			// Found caption
+			//fallback - failed to find a matching asset - fallback to first found if exist
+			if(source == null && this.textSources.length){
```

Comment:
@eitanavgil this will present a new behviour which might surprise anyone not expecting it.
Although it might be reasonable I prefer we add this from application side and not introduce new behviour here which is not expected.